### ISSUE TITLE
Browserstack Windows with Chrome 78.0 has no WebGL

### ISSUE THREAD

**Issue by dsmilkov** [tensorflow/tfjs#2272]
To unblock our CI tests, we are temporarily locking the Chrome version to 77.0 to fix browserstack WebGL failure on Windows on Chrome 78.0. This is a tracking issue to make sure we revisit and revert this fix later.

**Comment by gaikwadrahul8** [comment]
Hi, @dsmilkov 

Apologize for the delayed response and I see Google chrome browser compatibility matrix with WebGL https://caniuse.com/?search=webgl and latest versions are supporting WebGL support so it seems like we have taken care of this issue if I'm not wrong

CC :@pyu10055

If Am I missing something here please let me know ? Thank you!

**Comment by gaikwadrahul8** [comment]
Hi, @dsmilkov 

Thank you for opening this issue for tracking purposes. Since this issue has been open for a long time, the code/debug information for this issue may not be relevant with the current state of the code base.

The TFJs team is constantly improving the framework by fixing bugs and adding new features. We suggest you try the latest TFJs [version](https://github.com/tensorflow/tfjs/releases/tag/tfjs-v4.11.0) with the latest compatible hardware configuration which could potentially resolve the issue. We can keep the issue open if it is still relevant. Please confirm if we need to keep the issue open. 

Thank you for your support and cooperation.

**Comment by gaikwadrahul8** [comment]
Hi, @dsmilkov 

We have not received any confirmation from you, so we are closing this issue now. If the issue is still relevant, please let us know, either we'll re-open this issue or please feel free to create new issue after trying with latest version of Tensorflow.js. Thank you!
